lilypad/lilypad-aurora-testnet/overview.md

```diff
@@ -1,27 +1,63 @@
----
-cover: ../.gitbook/assets/aurora record player.png
-coverY: -9.129886506935662
-layout:
-  cover:
-    visible: true
-    size: full
-  title:
-    visible: true
-  description:
-    visible: true
-  tableOfContents:
-    visible: true
-  outline:
-    visible: true
-  pagination:
-    visible: true
----
-
-# Overview
-
-
-
-{% embed url="https://youtu.be/yQnB2Yxia4Y?si=x3EOrTy4M_cSao7c" %}
-Lilypad: Internet-scale Distributed Compute - web3's new frontier
-{% endembed %}
+  ---
+  cover: ../.gitbook/assets/aurora record player.png
+  coverY: -9.129886506935662
+  layout:
+    cover:
+      visible: true
+      size: full
+    title:
+      visible: true
+    description:
+      visible: true
+    tableOfContents:
+      visible: true
+    outline:
+      visible: true
+    pagination:
+      visible: true
+  description: Trustless Distributed Compute for web3
+  ---
 
+  # Overview
+
+  ### What is Lilypad?
+  Lilypad is a verifiable trustless decentralized compute network. It aims to enable mainstream web3 applications by providing global, permissionless access to compute power. Lilypad leverages decentralized physical infrastructure networks like Filecoin to create an open compute network that is more accessible, efficient, and transparent. It addresses the challenges surrounding access to AI models and hardware by offering decentralized AI capabilities. Lilypad has recently released Lilypad V2 (Aurora) and has plans to go multichain and launch an incentivized test net.
+
+  ### Objective and problem statement
+  Lilypad is aiming to solve the challenges surrounding access to AI models and hardware. Currently, there are significant barriers to entry for developers and organizations who want to leverage AI in their applications. Access to AI models and hardware is often limited, expensive, and centralized. Lilypad aims to address these issues by creating a verifiable trustless decentralized compute network that provides global, permissionless access to compute power. By leveraging decentralized physical infrastructure networks like Filecoin, Lilypad aims to make AI more accessible, efficient, and transparent. 
+
+  ### Applications
+  The ability to perform off-chain decentralised compute over data from smart contracts opens the door to a multitude of possible applications including:
```

Comment:
perform off-chain decentralised compute over data, with on-chain guarantees, and to call this functionality directly from a smart contract, CLI and an easy to use abstraction layer, opens the door to a multitude of possible applications including: